### DISCUSSION TITLE
테스트를 어디까지 해야 하나?

### DISCUSSION THREAD

**Discussion by AlbertImKr** [weaverse-techtide/weaverse-backend#102] [General]
테스트를 어디까지 해야 할까요?
테스트 코드가 점점 많이지고 있어요.

**Comment by Yoo117** [comment]
사실 이는 course에서 너무 많은 기능을 함축하고 있기에 생기는 거라고 볼 수도 있지 않을까요? 앱 기능을 분리해보는건 어떨지 조심스럽게 생각해 봅니다.
